### PR TITLE
Ck feature5 (co-authored w/ @josi543)

### PR DESCRIPTION
**Pull request by fxnggg**
### Feature 5
_co-authored w/ @josi543_ 

Tasks Completed:

- [x] Fix styling of previous Hackathon experience field to align with the rest of the form (feat. 1 style fix)
- [X] Add in starter projects subsection under resources
- [X] Add in appropriate links under starter projects
- [X] Add in icon and starter projects link on home page

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -254,7 +255,32 @@
...
+            "title": "Arduino",
+            "links":"<li> <a class=\"link\" href =\"https://docs.arduino.cc/\" target = _blank>Arduino Documentation</a></li><li> <a class=\"link\" href =\"https://github.com/arduino/\" target = _blank>Arduino Github Resources</a></li>"
+        },
+        "external_API_requests":{
+            "title": "External API Requests",
+            "links": "<li> <a class=\"link\" href =\"https://docs.xano.com/working-with-data/functions/external-api-request  \" target = _blank> What are External API Requests?</a></li><li> <a class=\"link\" href =\"https://github.com/topics/external-api  \" target = _blank>Github Resources </a></li>"
+        },
+        "security_configuration":{
+            "title": " Security Configurations",
+            "links": "<li> <a class=\"link\" href=\"https://www.sciencedirect.com/topics/computer-science/configuration-security \" target=_blank> What are Security Configurations? </a></li><li> <a class=\"link\" href =\" \" target = _blank> Github Repository</a></li>"
```
Comment (alison-nicole) [inline src/assets/i18n/en.json:269]: In the `Security Configuration` section, the  GitHub repository link is missing.

Comment (fxnggg) [reply src/assets/i18n/en.json:269]: @josi543 and I couldn’t find a good GitHub repo for it yet so we just have there as a placeholder

**Review by alison-nicole — COMMENTED** [review]
The only issue I see is that when the GitHub Repository link for the  `Security Configurations` section is clicked, it opens a local host instance of the page.

File: src/assets/i18n/en.json (hunk 1 above)
Comment (alison-nicole) [reply src/assets/i18n/en.json:269]: I thought so! In that case, Ignore my comment

Comment (fxnggg) [reply src/assets/i18n/en.json:269]: I’ll add it to our Trello board under to-do so it doesn’t get forgotten about

**Review by alison-nicole — APPROVED** [review]
Issue addressed. Everything looks great!

File: src/assets/i18n/en.json (hunk 1 above)
Comment (alison-nicole) [reply src/assets/i18n/en.json:269]: yes, thank you!

**Review by SonalChadha — APPROVED** [review]

**Pull request merged**